presto-kafka/pom.xml

```diff
@@ -68,14 +68,20 @@
         </dependency>
 
         <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>kafka-clients</artifactId>
+            <version>${dep.kafka-clients.version}</version>
+        </dependency>
+
+        <dependency>
             <groupId>joda-time</groupId>
             <artifactId>joda-time</artifactId>
         </dependency>
-
+        <!--
```

Comment:
Please delete rather than commenting out. The old version is available in the Git history. Commenting out code quickly clutters the code base.